modules/sc-mesh-secure-deployment/src/2_0/features/jamming/jamming_setup.py

```diff
@@ -21,9 +25,9 @@ def is_interface_up(interface) -> bool:
     return interface in available_interfaces
 
 
-def switch_frequency(args) -> None:
+async def switch_frequency(args) -> None:
```

Comment:
Is jamming avoidance running on top of upper macsec?

Reply:
Slack comments got from @TIISR to this. This interface is available also in new design.

Reply:
Yes and no: JA cannot be enabled by default, it needs to be enabled by user choice in MDM so it is a "red tier" feature in that sense, but JA won't be "running on top of upper macsec" in the sense of inspecting red network interfaces, it is meant to inspect black (or white in case we add something like HaLow there, as opposed to bat0 being the only white interface in most cases) network interfaces 🙂

Reply:
Sorry we were writing at the same time: my "yes and no" was in response to the top comment 🙂